app/src/test/scala/org/alephium/explorer/persistence/queries/InputQueriesSpec.scala

```diff
@@ -198,4 +198,28 @@ class InputQueriesSpec
       }
     }
   }
+
+  "index 'inputs_pk'" should {
+    "get used" when {
+      "accessing column output_ref_key" in {
+        forAll(Gen.listOf(inputEntityGen())) { inputs =>
+          run(InputSchema.table.delete).futureValue
+          run(InputSchema.table ++= inputs).futureValue
+
+          inputs foreach { input =>
+            val query =
+              sql"""
+                   |SELECT *
+                   |FROM inputs
+                   |where output_ref_key = ${input.outputRefKey}
+                   |""".stripMargin
+
+            val explain = run(query.explain()).futureValue.mkString("\n")
+
+            explain should include("Index Scan using inputs_pk on inputs")
```

Comment:
all those tests are so great :rocket:

Reply:
I guess this test case is flaky as well. Build on master is [failing](https://github.com/alephium/explorer-backend/runs/7497172580?check_suite_focus=true#step:5:592). 

Will make changes according to the outcome #292.